py/torch_tensorrt/dynamo/fx_ts_compat/lower.py

```diff
@@ -198,7 +200,7 @@ def default_split_function(
     model: fx.GraphModule, inputs: Input, lower_setting: LowerSetting
 ) -> SplitResult:
     splitter_setting = TRTSplitterSetting()
-    splitter_setting.use_implicit_batch_dim = not lower_setting.explicit_batch_dimension
+    splitter_setting.use_implicit_batch_dim = False
```

Comment:
Since `explicit_batch_dimension` is no longer an allowed input argument due to deprecation, we set this to False.